pancloud/adapters/tinydb_adapter.py

```diff
@@ -32,7 +32,7 @@ def fetch_credential(self, credential=None, profile=None):
         """
         q = self.db.search(self.query.profile == profile)
         try:
-            return q[0].get(credential, '')
+            return q[0].get(credential, None)
         except (AttributeError, ValueError, IndexError):
```

Comment:
Can this be reduced to just IndexError?

Reply:
Yes, it does seem a bit paranoid. The "exception creep" happened during the development process, which I intended to clean up. I'll make the change.